module6/memory-tests-v1/memory-tests-main/java7-test5.bat

```diff
@@ -0,0 +1 @@
+cd target/classes/ & D:\work\tools\Java\jdk1.7.0_79\bin\java -XX:ThreadStackSize=1024 by.epam.jamp.module6.main.configs.Main 
```

Comment:
to note:
There is also *Xss* option that should act as *XX:ThreadStackSize* but on various VM they act differently.